freight/providers/shell.py

```diff
@@ -13,12 +13,14 @@ def get_options(self):
         }
 
     def get_command(self, task, ssh_key):
+        params = task.params or {}
+
         return task.provider_config['command'].format(
             environment=task.environment,
             sha=task.sha,
             ref=task.ref,
-            task=task.name,
             ssh_key=ssh_key,
+            **params
```

Comment:
we might want to just leave it as params=params

in the formatter you can always do "params.foo" IIRC